assets/js/state/selectors/index.js

```diff
@@ -68,6 +69,20 @@ const enrichInstances = (instances, sapSystemId, state) =>
       };
     });
 
+export const findInstancesOnHost = (hostId) => (state) => {
```

Comment:
why find and not `get` ?

Reply:
my bad, I broke consistency :sweat_smile:  
Gonna normalize those in `getSomething`.

Reply:
Changed here https://github.com/trento-project/web/pull/244/commits/6dba40e69436bf0ac5af3ade780889ffd7788912